storybook/storybook-docs/src/grid.stories.mdx

```diff
@@ -4,110 +4,153 @@ import { Meta } from "@storybook/blocks";
+Dat lijkt misschien onnatuurlijk, maar is geen probleem.
+Elementen bezetten vaak meerdere kolommen van het grid en krijgen dan ook de tussenliggende witruimtes mee.
 
 #### Compact
 
 Voor applicaties is zo veel witruimte niet nodig, zelfs contraproductief.
-Daarom is er een compact thema voor het grid.
+Daarom is er ook een compact thema voor het grid.
 
-Hier begint de horizontale witruimte op 4 pixels bij een vensterbreedte van 320 pixels,
-en groeit deze met 4 pixels per 256 pixels extra breedte.
-Het maximum is hier 40 pixels en dat wordt pas bereikt bij een vensterbreedte van 2.624 pixels.
+De minimale witruimte tussen kolommen is ook hier 16 pixels.
+Maar deze begint pas op een vensterbreedte van 1088 pixels te groeien.
+De toename gaat ook langzamer: 4 pixels per 256 pixels extra breedte.
+(Dit komt neer op 1 en 9/16 procent van de schermbreedte min 1 pixel.)
+Het maximum is hier 40 pixels en dat wordt pas bereikt bij een vensterbreedte van 2624 pixels.
 
-Alle andere eigenschappen van de compacte variatie zijn gelijk aan de ruimtelijke.
+Alle andere eigenschappen van het compacte thema zijn gelijk aan die van het ruimtelijke.
 
 ### Marges links en rechts
 
-#### Voor websites
-
-Het grid centreert zichzelf horizontaal.
+Het grid staat horizontaal gecentreerd.
 Aan beide zijden reserveert het grid marges om afstand te houden tot de randen van het venster.
-Deze ruimte is 1½ keer zo breed als die tussen de kolommen.
 
-Sommige elementen mogen over deze marges gepositioneerd worden, zoals de Page Footer en een schermvullende afbeelding.
-Die zijn dan dus breder dan de rest van de content, maar worden wel beperkt door een maximale breedte.
+In het ruimtelijke thema is deze ruimte 1½ keer zo breed als die tussen de kolommen.
+Sommige elementen mogen over deze marges gepositioneerd worden, zoals de pagina-footer en een schermvullende afbeelding.
+Die zijn dan dus breder dan de rest van de content.
+Ze worden nog wel beperkt door een maximale breedte.
 
-Voor applicaties is de marge naast het grid gelijk aan de witruimtes erbinnen.
+In het compacte thema zijn de marges buiten het grid even breed als de witruimtes erbinnen.
 Hier worden geen elementen over de marges heen geplaatst.
 
-### Niet onbeperkt breed
+### Menubalk
 
-Een maximale breedte voor het grid zorgt ervoor dat de elementen van een pagina niet al te ver van elkaar verwijderd raken.
-Ook wordt de witruimte en typografie dan niet extreem groot.
+Applicaties hebben in het brede grid een verticale menubalk aan de linkerzijde.
+Deze staat naast het grid, niet erin, en heeft een vaste breedte van 112 pixels.
 
-Voor websites is de maximale breedte van het grid 1432 pixels.
-In vensters van minimaal 1600 pixels breed heeft het grid deze breedte – de marges bedragen hier 84 pixels.
+In het middelbrede en smalle grid wordt het menu verticaal.
```

Comment:
Not for applications right? It will be hard to read if the application theme and spacious theme are mixed up.

Reply:
There is no mixing up going on here.

This is about the blue menu bar for applications, not the ‘mega menu’ of websites. The menu bar for applications has a horizontal variation that is placed below the header on narrower screens. So I did type the orientation incorrectly, changed it to ‘horizontal’.